code/modules/power/battery.dm

```diff
@@ -225,8 +225,8 @@ var/global/list/battery_online =	list(
 	var/data[0]
 	data["nameTag"] = name_tag
 	data["storedCapacity"] = round(100.0*charge/capacity, 0.1)
-	data["charge"] = charge
-	data["capacity"] = capacity
+	data["charge"] = charge/SMESRATE //Compensates for the input/output rate
```

Comment:
Shouldn't this be times, not divided...?

Reply:
SMESRATE is 0.05 and acts as a divisor and multiplier on the charge that gets in and out of the SMES. The lower it is the more power the SMES can store with the same numbers as it multiplies the power that gets in through SMESRATE and divides the power that gets out.